users/harterj/index.md

```diff
@@ -34,9 +34,7 @@ We have shown Rattlesnake to be an effective and efficient engine for simulating
   * Higher-order scattering moments to model anharmonic phonon interactions
 * Multi-frequency phonon transport equation in SAAF form with implicit temperature dependence in equilibrium phonon distribution
 $$
-\begin{align*}
--\Lambda\left(\omega,p\right) \vec{\Omega}\cdot \left[\Lambda\left(\omega,p)\vec{\Omega}\cdot \vec{\nabla} \psi\left(\vec{r},\vec{\Omega},\omega,p\right)\right] + \psi\left(\vec{r},\vec{\Omega},\omega,p\right) = -\Lambda\left(\omega,p\right)\vec{\Omega}\cdot \vec{\nabla}\phi^{0}\left(\vec{r},\omega,T,p\right) + \phi^{0}\left(\vec{r},\omega,T,p\right)
-\end{align*}
+-\Lambda\left(\omega,p\right) \vec{\Omega}\cdot \left[\Lambda\left(\omega,p\right)\vec{\Omega}\cdot \vec{\nabla} \psi\left(\vec{r},\vec{\Omega},\omega,p\right)\right] + \psi\left(\vec{r},\vec{\Omega},\omega,p\right) = -\Lambda\left(\omega,p\right)\vec{\Omega}\cdot \vec{\nabla}\phi^{0}\left(\vec{r},\omega,T,p\right) + \phi^{0}\left(\vec{r},\omega,T,p\right)
```

Comment:
@rtrp/admins Yeah. So the deal, is, the double dollar sign to start and end latex, single dollar sign for in-line latex. I would say always check your code in texworks or whatever to make sure it doesn't fail before trying to push. The reason we can use latex is in the _config.yml file, where you see `markdown: kramdown` is the markdown set where latex is possible. I am pretty sure I set that up ages ago but forgot, but kramdown is the setting to use for latex.